perun-core/src/main/java/cz/metacentrum/perun/core/entry/ResourcesManagerEntry.java

```diff
@@ -653,7 +677,7 @@ public int getResourcesCount(PerunSession sess) throws InternalErrorException {
 
 		// Authorization
 		if (!AuthzResolver.isAuthorized(sess, Role.PERUNADMIN) && !AuthzResolver.isAuthorized(sess, Role.VOADMIN, member)
-				&& !AuthzResolver.isAuthorized(sess, Role.ENGINE)
+				&& !AuthzResolver.isAuthorized(sess, Role.ENGINE) && !AuthzResolver.isAuthorized(sess, Role.PERUNOBSERVER)
```

Comment:
We can omit check on PERUNADMIN role here, since any of the check will be true for such user. Check on PERUNOBSERVER stays of course.